doc/design/service-discovery.md

```diff
@@ -340,7 +332,7 @@ StatusPortReader {
 };
 
 // User facing API
-class DiscoveryInfo
+class ServiceDiscovery
```

Comment:
Why adding an implementation for an alternative which we are currently not pursuing?

Reply:
For readers to see the differences to the other alternatives right away and not confuse readers with naming differences which are irrelevant. Think of our futures selves which want to understand what happened in 2022 :wink: